jenkins-install/start-docker-compose.bash

```diff
@@ -0,0 +1,7 @@
+cd ~/ncdevopsschool22/jenkins-install
```

Comment:
This path (~) may not work for others, because we assume that code cloned to home directory. I suggest to move instructions how to build and how to start Jenkins in Readme